tracing/tracer.go

```diff
@@ -37,25 +27,67 @@ func (tp *TestTraceProvider) Tracer(name string) trace.Tracer {
 var Configured bool
 
 type Config struct {
+	ServiceName string            `long:"service-name"  description:"otlp service name" default:"concourse-web"`
```

Comment:
this one is not otlp-specific, so maybe something like
```suggestion
	ServiceName string            `long:"service-name"  description:"service name to attach to traces as metadata" default:"concourse-web"`
```